pkg/apis/work/v1alpha2/well_known_labels.go

```diff
@@ -25,6 +25,9 @@ const (
 
 	// WorkNameLabel is added to objects to specify associated Work's name.
 	WorkNameLabel = "work.karmada.io/name"
+
+	// ObjectReferenceUIDLabel is added to objects to specify object's uid
+	ObjectReferenceUIDLabel = "objectference.k8s.io/uid"
```

Comment:
It's not a label I guess.

Reply:
Yes. It is not good.
`ClusterResourceBindingLabel`, `ResourceBindingNamespaceLabel` and `ResourceBindingNameLabel` is annotation in this file

Reply:
Thanks for reminding me.  Yeah, I'll optimize it by a separate PR.

Reply:
opened #1979 for this.